pylib/cqlshlib/cqlshmain.py

```diff
@@ -967,7 +968,10 @@ def perform_simple_statement(self, statement):
         if not statement:
             return False, None
 
-        future = self.session.execute_async(statement, trace=self.tracing_enabled)
+        query_to_execute = statement.query_string
+        if self.filtering_enabled:
+            query_to_execute = statement.query_string[:-1] + " ALLOW FILTERING;"
```

Comment:
would this be added to like ALTER and DESCRIBE commands?

Reply:
@clohfink possibly to ALTER, hardly to DECRIBE as that is kind of done differently as we are just getting what Cassandra gives us ... well ... I think that returning to having a property (not schema property!) on TableMetadata and tweaked comment about enabling ALLOW FILTERING is just better after all.

BTW I would not modify comments for each vtable being able to be filtered on. That would be a default state. We would mention that only if _ALLOW FILTERING is not possible_.